src/EnergyPlus/DXCoils.cc

```diff
@@ -12148,7 +12155,7 @@ Label50: ;
 		Real64 PartLoadAirMassFlowRate;
 		Real64 AirMassFlowRatio;
 		static Real64 AccuracyTolerance( 0.2 ); // tolerance in AHRI 340/360 Table 6 note 1
-		static int MaximumIterations( 500 );
+		static int MaximumIterations( 1000 );
```

Comment:
What I saw in the debugger was that in RegulaFalsi the lower bound of mass flow rate at 0.01 * full mass flow gave a supply air temperature of -1000 and full mass flow rate showed about 14C SAT. With a target SAT of about 12C, RegulaFalsi thought the solution was very near full mass flow rate and took very small steps towards the final answer. It was getting close at 500 iterations but needed more to get there. Could probably do a better job of selecting the bounds for the RegulaFalsi call.